ts/twitter/src/app/home/__test__/home.spec.tsx

```diff
@@ -42,6 +42,8 @@ jest.mock("next/navigation", () => ({
 
 describe("Home Tests", () => {
   test("Rendering Home should success", () => {
+  // TODO: https://github.com/okuda-seminar/Twitter-Clone/issues/522
+  // - Fix test descriptions in home.spec.tsx and post-modal.tsx.
```

Comment:
nit: Let's move the TODO comment to above the issue point.

```suggestion
  // TODO: https://github.com/okuda-seminar/Twitter-Clone/issues/522
  // - Fix test descriptions in home.spec.tsx and post-modal.tsx.
  test("Rendering Home should success", () => {
```